src/components/MarkdownProvider/components/CodeExample/assets/codesandbox-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16">
+  <path fill="currentColor" fill-rule="evenodd" clip-rule="evenodd" d="M8 0c-.35 0-.695.092-.998.266H7L1.753 3.255l-.002.001A1.997 1.997 0 00.75 4.981v5.977a1.99 1.99 0 001 1.726h.003L7 15.673a2.005 2.005 0 001.997 0H9l5.248-2.987.002-.001a1.997 1.997 0 001.001-1.726V4.98a1.99 1.99 0 00-1-1.725L9 .267h-.002A2.005 2.005 0 008 0zm-.5 1.134a1.005 1.005 0 011 0l1.83 1.04L8 3.515l-2.33-1.34 1.827-1.04.002-.001zM4.662 2.748L2.28 4.104 8.001 7.4l5.719-3.296-2.383-1.356L8.25 4.525a.5.5 0 01-.498 0L4.663 2.748zM1.75 4.981v-.029L7.5 8.265v6.54l-2.375-1.351V9.912a.5.5 0 00-.25-.434L1.75 7.68V4.981zm6.75 9.825l.003-.002 2.372-1.35V9.912a.5.5 0 01.25-.434L14.25 7.68V4.982v-.029L8.5 8.266v6.54zm5.75-5.972l-2.375 1.367v2.683l1.876-1.067a.997.997 0 00.499-.86V8.834zm-12.5 2.123V8.834l2.375 1.367v2.683l-1.876-1.067a.997.997 0 01-.499-.86z" />
```

Comment:
In keeping with `svg-icons` optimizations, you should be able to delete `fill-rule` and `clip-rule`.

Reply:
I thought so too, but this is what I get without it:
![image](https://user-images.githubusercontent.com/29072694/89458883-a7a6bd00-d71c-11ea-9adf-6f5d9ef776e2.png)

Reply:
I'm gonna try reflattening in Sketch, one sec.

Reply:
Was able to omit `clip-rule` on re-flattening, but if `fill-rule` is missing it renders like the above.

Reply:
@rossmoody want to send me your Abstract? I can help troubleshoot.